ecs.tf

```diff
@@ -4,6 +4,7 @@ data "aws_ecs_cluster" "target_cluster" {
 
 resource "aws_ecs_task_definition" "main_task" {
   family                = "${var.app_name}-tsk"
+  requires_compatibilities = [var.launch_type]
```

Comment:
What happens if `var.launch_type` is null?

Reply:
`var.launch_type` has a default value of "EC2".

Reply:
Then you messed up. A default is provided.

Reply:
And if `requires_compatibilities` wasn't included would the default be `["EC2"]`?

Reply:
Just otherwise wouldn't it be a breaking change _technically_? Don't know if it will have an effect anywhere down stream, but also don't know if it _won't_ have an effect downstream.